plugins/primus_lisp/primus_lisp_primitives.ml

```diff
@@ -300,6 +322,8 @@ module Primitives(Machine : Primus.Machine.S) = struct
         "(memory-read A) loads one byte from the address A";
       def "memory-write" (tuple [int; byte] @-> int) (module MemoryWrite)
         "(memory-write A X) stores by X to A";
+      def "memory-fill" (tuple [int; int; byte] @-> int) (module MemoryFill)
+        "(memory-fill A L X) fills every byte from A to A + L with X";
```

Comment:
the type signature is made incorrect, as you return an 8-bit value (aka byte), not a word. But don't fix the signature, fix the implementation - let it return `A` and update the documentation to reflect this.